code/_onclick/other_mobs.dm

```diff
@@ -89,6 +89,8 @@
 	for(var/datum/mutation/human/HM in dna.mutations)
 		HM.on_ranged_attack(src, A, mouseparams)
 
+	SendSignal(COMSIG_HUMAN_RANGED_ATTACK, A)

```

Comment:
Make sure to update your branch to latest, this method of calling on signals is outdated now.

Reply:
shooooooot